dags/main_summary.py

```diff
@@ -398,6 +398,14 @@
     executor=GetDefaultExecutor(),
     dag=dag)
 
+exact_mau_by_client_count_dimensions = bigquery_etl_query(
+    task_id="exact_mau_by_client_count_dimensions",
+    destination_table="firefox_desktop_exact_mau28_by_client_count_dimensions_v1",
+    dataset_id="telemetry_derived",
+    owner="mreid@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "mreid@mozilla.com"],
```

Comment:
I'd encourage you to make me or relud owner on these, as they're so similar to the existing exact_mau tables and I'd argue you should avoid dataset ownership in preference to management duties. I volunteer to be listed as owner.

Reply:
I was going to ask this very question. I will update it in a follow-up.